port/pool.go

```diff
@@ -0,0 +1,46 @@
+package port
+
+import (
+	log "github.com/cihub/seelog"
+	"math/rand"
+	"time"
+)
+
+type Port struct {
+	Number int
+}
+
+func (p *Port) Release() {
+	log.Debug("releasing port back to pool: ", p.Number)
+}
+
+type Pool struct {
+	start, cap int
```

Comment:
Do not abbreviate in structs, unless its acronym or very know abbreviation. Cap may mean max, top not only amount. `capacity` not that long word after all  :)